<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Page">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>911</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>731</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_code">
           <property name="text">
            <string>Participant Code: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_code"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_trial">
           <property name="text">
            <string>Trial:                     #</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_trial"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="auto_complete">
           <property name="text">
            <string>Auto</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">button_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="overwritten">
           <property name="text">
            <string>Overwriten</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">button_group</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="start">
         <property name="text">
          <string>start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="m_countdownLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>450</y>
        <width>531</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_duration">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>390</y>
        <width>729</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Dynamic</string>
     </attribute>
     <widget class="QCustomPlot" name="backdrop" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>461</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="show">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>230</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>show</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>170</y>
        <width>190</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_filepath">
         <property name="text">
          <string>File Path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filepath"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="exit">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>280</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="button_group"/>
 </buttongroups>
</ui>
